tools/ci/setup_docker_dataverse.sh

```diff
@@ -3,9 +3,9 @@ set -e -u
 # This is setting up and launching a docker container with everything needed for
 # our tests to run against this instance.
 # Most importantly with respect to the CI builds, it deposits the API tokens
-# for the users `pete` and `uma` in environment variabes TEST_TOKEN_PETE and
-# TEST_TOKEN_UMA respectively for use from within unit tests. Therefore this
-# script is supposed to be sourced.
+# for the users `pete` and `uma` in environment variabes
```

Comment:
Phew, only now noticed there are `pete` and `uma` still in that comment (that's from the demo script that comes with the container).
Should be `testadmin` and `user1` now. Doesn't need to be part o this PR, but seems like an occasion to clean up.

Reply:
If you provide a concrete suggestion, it would be possible with a click. Otherwise I would need to learn what pete and uma are and what exactly needs changing -- which I would file an issue for and do a lot later.

Reply:
I'll push - gimme a minute.

Reply:
OK, that was straightforward. Thanks!